src/pymodaq/extensions/daq_scan.py

```diff
@@ -887,11 +888,11 @@ def start_scan(self):
             # mandatory to deal with multithreads
             if self.scan_thread is not None:
                 self.command_daq_signal.disconnect()
-                if self.scan_thread.isRunning():
+                self.scan_thread.quit()
```

Comment:
same as for the control modules remove this out from here into a dedicated branch (you can cherry pick the corresponding commit if this modifcation has been done alone)